scripts/build_salt_cavern_potentials.py

```diff
@@ -0,0 +1,504 @@
+# -*- coding: utf-8 -*-
+"""
+Build salt cavern potentials for hydrogen storage.
+
+https://doi.org/10.3133/sir20105090S
+"""
+
+import functools
+import math
+import os
+import shutil
+import zipfile
+from pathlib import Path
+
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+import rasterio.features
+import requests
+import rioxarray
+import shapely.geometry
+from _helpers import (
+    mock_snakemake,
+    to_csv_nafix,
+)
+
+
+def download_potash_data():
+    # URL of the Potash GIS data
+    url = "https://pubs.usgs.gov/sir/2010/5090/s/PotashGIS.zip"
```

Comment:
Good to have a TODO: to potentially externalize the download of the files in a future retrieve_* rule. No need now but worth tracking with a TODO

Reply:
I'd expect this function has some duplications with some functions of the helper and the databundle but probably it is not worth the effort now